libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -181,29 +182,38 @@ public UserIdentity generateUserIdentity(
             Optional<JsonNode> ninoClaim = generateNinoClaim(successfulVCStoreItems);
             ninoClaim.ifPresent(userIdentityBuilder::ninoClaim);
 
-            userIdentityBuilder.exitCode(getSuccessExitCode(contraIndicators));
+            userIdentityBuilder.returnCode(getSuccessReturnCode(contraIndicators));
         } else {
-            userIdentityBuilder.exitCode(getFailExitCode(contraIndicators));
+            userIdentityBuilder.returnCode(getFailReturnCode(contraIndicators));
         }
 
         return userIdentityBuilder.build();
     }
 
-    private List<String> getFailExitCode(ContraIndicators contraIndicators)
+    private List<ReturnCode> getFailReturnCode(ContraIndicators contraIndicators)
             throws UnrecognisedCiException {
         return ciMitUtilityService.isBreachingCiThreshold(contraIndicators)
-                ? mapCisToExitCodes(contraIndicators)
-                : List.of(configService.getSsmParameter(EXIT_CODES_NON_CI_BREACHING_P0));
+                ? mapCisToReturnCodes(contraIndicators)
+                : List.of(
+                        new ReturnCode(
+                                configService.getSsmParameter(RETURN_CODES_NON_CI_BREACHING_P0)));
     }
 
-    private List<String> getSuccessExitCode(ContraIndicators contraIndicators)
+    private List<ReturnCode> getSuccessReturnCode(ContraIndicators contraIndicators)
             throws UnrecognisedCiException {
-        return mapCisToExitCodes(contraIndicators).stream()
-                .filter(configService.getSsmParameter(EXIT_CODES_ALWAYS_REQUIRED)::contains)
+        List<ReturnCode> returnCodes = mapCisToReturnCodes(contraIndicators);
+        return returnCodes.stream()
+                .filter(
+                        returnCode ->
+                                returnCode
+                                        .code()
+                                        .equals(
```

Comment:
Should this be equals, or contains?

Reply:
Ah yep, good catch. It would be fine for now, but if we ever add more it'll fail. I'll update.